token-lending/program/tests/helpers/mod.rs

```diff
@@ -515,7 +515,7 @@ impl TestLendingMarket {
             Some(&payer.pubkey()),
         );
 
-        let recent_blockhash = banks_client.get_recent_blockhash().await.unwrap();
+        let recent_blockhash = banks_client.get_latest_blockhash().await.unwrap();
```

Comment:
Changes in this file are to deprecate get_recent_blockhash in favor of get_latest_blockhash, a result of upgrading to 1.9.5.